database/migrations/2025_02_15_135543_create_posts_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
```

Comment:
`$table->timestamps();` should be the last statement of this function. Changing the order causes the columns appear in a different order when you check the table in a database management tool.